app/admin/books.rb

```diff
@@ -1,8 +1,21 @@
 ActiveAdmin.register Book do
 
   menu :priority => 4
+  sidebar :versions, :partial => "admin/version", :only => :show
 
-  index do 
+  controller do
+    def create
+      # We have books already in the db for which paper_trail will not create versions.
+      # After it is installed paper_trail creates an initial version for each record created.
+      # We turn that off here so that version 1 of a new record means
+      # the same thing as version 1 of an old record (i.e. the first time the record was updated).
+      PaperTrail.enabled = false
+      super
+      PaperTrail.enabled = true
```

Comment:
We don't want to save a version when the object is created through the admin interface.  This keeps it consistent with exiting records, because Paper Trail doesn't create a version for the creation of records that already exist.